src/sysfiles.c

```diff
@@ -711,41 +774,67 @@ const Char * SyWinCmd (
 */
 
 
-/****************************************************************************
-**
-*V  syBuf . . . . . . . . . . . . . .  buffer and other info for files, local
-**
-**  'syBuf' is  a array used as  buffers for  file I/O to   prevent the C I/O
-**  routines  from   allocating their  buffers  using  'malloc',  which would
-**  otherwise confuse Gasman.
-**
-**
-**  Actually these days SyBuf just stores various file info. SyBuffers
-**  stores buffers for the relatively few files that need them.
-*/
-
-SYS_SY_BUF syBuf [256];
-
-SYS_SY_BUFFER syBuffers [ 32];
-
 // Mark a member of syBuf as unused
-void SyMarkBufUnused(Int i)
+static void SyBufMarkUnused(Int i)
 {
     GAP_ASSERT(i >= 0 && i < 256);
     memset(&(syBuf[i]), 0, sizeof(syBuf[i]));
     syBuf[i].type = unused_socket;
+    syBuf[i].fp = -1;
```

Comment:
This is not a requirement, but I wonder if we should just do a memset here(ISH), to blank the buffer back to initial settings (before setting unused_socket, and fp=-1)

Reply:
There already is an memset - perhaps I misunderstand?

Reply:
You don't misunderstand, I just somehow missed it. Woops.